api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json

```diff
@@ -18,8 +22,11 @@
       "adults": 4,
       "children": 2,
       "message": "Important message",
+      "locked": null,
       "created_at": "CREATED_AT",
       "updated_at": "UPDATED_AT"
     }
-  ]
+  ],
+  "meta": {
```

Comment:
Hmm true, no meta for this.
Regarding other responses, just updated what I found. But they look identical. If anything when creating inquiry with `inquiries_write_owned` it locks inquiry, and returns oly inquiries locked by this application. So the response I gave makes no sense :P